storage-proofs-porep/tests/stacked_circuit.rs

```diff
@@ -177,5 +184,7 @@ fn test_stacked_porep_circuit<Tree: MerkleTreeTrait + 'static>(
         "inputs are not the same length"
     );
 
-    cache_dir.close().expect("Failed to remove cache dir");
+    if std::fs::remove_dir(cache_dir.path()).is_ok() && cache_dir.path().exists() {
+        let _ = cache_dir.close();
+    }
```

Comment:
Pretty sure I tried that and it didn't work (would sometimes error on the second line).

Reply:
Also, if you can find a different way to solve it, it would be appreciated.  The goal is just for it to 1) not cause an error, and 2) be gone after the run.  Took me a while to get that going and I agree it's weird (but works).

Reply:
Oh, that is weird. If you've already tried alternatives to the `if` and the tests' temp-files weren't fully removed, then the PR looks good to me.